### PR TITLE
smallrye-config-source-file-system allow multiple locations (#571)

### PR DESCRIPTION
**Pull request by henrikcaesar**

### CODE REVIEW

**Comment by henrikcaesar** [comment]
@radcortez Gave it a go, please review and let me know if anything needs to be changed

File: sources/file-system/src/main/java/io/smallrye/config/source/file/FileSystemConfigSourceFactory.java [hunk 1]
```diff
@@ -10,16 +12,17 @@
...
     @Override
     public Iterable<ConfigSource> getConfigSources(final ConfigSourceContext context) {
-        final ConfigValue value = context.getValue(SMALLRYE_CONFIG_SOURCE_FILE_LOCATION);
+        final ConfigValue value = context.getValue(SMALLRYE_CONFIG_SOURCE_FILE_LOCATIONS);
         if (value == null || value.getValue() == null) {
             return Collections.emptyList();
         }
-
-        return Collections.singletonList(new FileSystemConfigSource(value.getValue()));
+		return Stream.of(value.getValue().split(","))
```
Comment (radcortez) [inline sources/file-system/src/main/java/io/smallrye/config/source/file/FileSystemConfigSourceFactory.java:23]: Try using here `newArrayConverter(STRING_CONVERTER, String[].class).convert(value.getValue())`

**Review by radcortez — CHANGES_REQUESTED** [review]

**Comment by radcortez** [comment]
Thanks for the PR.

Left a suggestion to improve. Please, run the build locally to format the code.

File: sources/file-system/src/main/java/io/smallrye/config/source/file/FileSystemConfigSourceFactory.java (hunk 1 above)
Comment (radcortez) [reply sources/file-system/src/main/java/io/smallrye/config/source/file/FileSystemConfigSourceFactory.java:23]: Ops. I guess it is ok to add a new method to retrieve the a Converter by Type.

Comment (henrikcaesar) [reply sources/file-system/src/main/java/io/smallrye/config/source/file/FileSystemConfigSourceFactory.java:23]: Ok will change it next week

**Comment by radcortez** [comment]
Hi @henrikcaesar,

Are you going to update the PR? :) If not I can do it.

**Comment by henrikcaesar** [comment]
Hi @radcortez I haven't had time yet... if you can fix it I don't mind :)

**Comment by radcortez** [comment]
It's fine, we can wait for you :)

**Comment by henrikcaesar** [comment]
@radcortez fixed and hopefully correct formatting also

**Review by radcortez — APPROVED** [review]

**Comment by radcortez** [comment]
Thanks!

**Pull request merged**